packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -264,6 +245,28 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
     }
   }
 
+  private setActiveElement(element: HTMLElement): void {
```

Comment:
Private methods should be prefixed with `_`, i.e. `_setActiveElement`.

Reply:
Fixed.